dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/GeneralTestDataProvider.kt

```diff
@@ -28,6 +28,23 @@ class GeneralTestDataProvider {
         )
     }
 
+    fun generateCompanyInformationWithNameAndIdentifiers(
+        lei: String?,
+        isin: List<String>?,
+        permId: String?,
+    ): CompanyInformation {
+        var identifiers = emptyMap<String, List<String>>()
+        if (!lei.isNullOrEmpty()) identifiers = identifiers + mapOf(IdentifierType.lei.value to listOf(lei))
+        if (!isin.isNullOrEmpty()) identifiers = identifiers + mapOf(IdentifierType.isin.value to isin)
```

Comment:
Why is it listOf(lei) and then just isin?

Reply:
@PaulChr99

Reply:
See my comment in the ApiAccessor. They are handled differently due to the underlying business logic. I adapted the misleading input name to "isins" (plural)